xrviz/example.py

```diff
@@ -0,0 +1,39 @@
+from .dashboard import Dashboard
+
+
+def example(show=True):
+    """
+    Generates interface for the sample dataset.
+
+    Parameters
+    ----------
+    show: bool (True)
+        Whether to directly execute the interface. If True, a new browser tab
+        will be opened, and the function will block until interrupted. If
+        False, the Dashboard instance is returned without being executed.
+    """
+    from .sample_data import great_lakes
+    initial_params = {'Variables': 'temp',  # Select Variable
```

Comment:
Yes, I think this is exactly how I imagined it. We will have to describe in the docs, in detail, the set of parameters that can show up here.